ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py

```diff
@@ -688,3 +689,48 @@ def test_torch_from_dlpack(
         fn_tree=fn_tree,
         on_device=on_device,
     )
+
+
+@st.composite
+def _get_dtype_buffer_count_offset(draw):
+    dtype, value = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+        )
+    )
+    value = np.array(value)
+    length = value.size
+    value = value.tobytes()
+
+    offset = draw(helpers.ints(min_value=0, max_value=length - 1))
+    count = draw(helpers.ints(min_value=-(2**30), max_value=length - offset))
+    if count == 0:
+        count = -1
+    offset = offset * np.dtype(dtype[0]).itemsize
```

Comment:
For setting up the `offset: [int]`, it needs to be multiplied with the element size of the `dtype` object generated during the test. While Numpy offers this functionality to extract the itemsize from the dtype, I could not find the equivalent way to do so in Ivy.